src/pages/mypage/utils/analytics.ts

```
@@ -0,0 +1,274 @@
+ * - mypage_click_tabFurniture
+ * - mypage_click_btnImgCard
+ * - mypage_click_btnFurnitureCard
+ * - mypage_click_btnMakeImg
+ * - mypage_click_btnLogout
+ * - mypage_click_btnSuccession
+ * - mypage_click_successionModalOut
+ * - mypage_click_successionModalCancel
+ *
+ */
+
+import { logEvent } from 'firebase/analytics';
```

ESLint

[error] Replace `'firebase/analytics'` with `"firebase/analytics"`

(prettier/prettier)